docs/diagnostics/overview.md

```diff
@@ -13,12 +13,12 @@ Several APIs of .NET Aspire are decorated with the <xref:System.Diagnostics.Code
 
 .NET Aspire provides various overloads for Azure Provisioning resource types (from the `Azure.Provisioning` package). The overloads are used to create resources with different configurations. The overloads are experimental and may be removed or changed in future versions of .NET Aspire.
 
-To suppress this diagnostic with the `SuppressMessageAttribute`, add the following code to your project:
+To suppress this diagnostic with the `SuppressMessageAttribute`, add the following code to your project file:
 
-```csharp
-using System.Diagnostics.CodeAnalysis;
-
-[assembly: SuppressMessage("AZPROVISION001", "Justification")]
+```xml
+<PropertyGroup>
+  <NoWarn>$(NoWarn);ASPIREHOSTINGPYTHON001</NoWarn>
```

Comment:
1. Why is this using `ASPIREHOSTINGPYTHON001` in the `AZPROVISION001` section?
2. `AZPROVISION001` is no longer used and this section can be removed.

Reply:
1- Fixed, I updated the wrong section
2- Could it still be used in a previous version? At what point should we remove the related docs?

Reply:
I think we need to leave it in learn.microsoft.com. I'm not sure how long it should stay there, at least for a while.

In the `main` branch of the repo... I'd think it could be removed now, but don't have a strong opinion.